lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -210,6 +219,21 @@ class DetailsRenderer {
     return element;
   }
 
+  /**
+   * @param {LH.Audit.Details.List} details
+   * @returns {Element}
+   */
+  _renderList(details) {
+    const listContainer = this._dom.createElement('div', 'lh-list');
+
+    details.items.forEach(item => {
+      // @ts-ignore TODO(bckenny): this can never be null
+      listContainer.appendChild(this.render(item));
```

Comment:
rather than call `this.render(item)`, just call `SnippetRenderer.render()` directly (then also don't have to deal with `null`)